plugins/cursecheck.cpp

```diff
@@ -244,29 +193,23 @@ command_result cursecheck (color_ostream &out, vector <string> & parameters)
                 }
```

Comment:
I think you can replace this entire `if unit->name.has_name ... else` block with `out.print(Units::getReadableName(unit);`

Reply:
Thanks, I was looking for this functionality but didn't find it! :+1:

Reply:
it's relatively new. I got tired of synthesizing a "descriptive unit string" in multiple tools and finally created a new centralized API for it